patch.py

```diff
@@ -93,15 +103,18 @@ def getName(value):
         if (status == 1):
             print('1 line under GENERAL updates:')
             print('* ' + ' '.join(lines))
-            print('\nThis line might be a hero/item update and you should manually place it at the proper location.')
+            print('\nThis line might be a hero/item update and you ' + \
```

Comment:
__[flake8]__
* [E502] the backslash is redundant between brackets

<!-- Comment by [SideCI](https://sideci.com) -->